README.md

```diff
@@ -121,6 +121,33 @@ Devnet predeploys a [UDC](https://docs.openzeppelin.com/contracts-cairo/0.6.1/ud
 
 For now, you can consult the [Pythonic Devnet docs on minting](https://0xspaceshard.github.io/starknet-devnet/docs/guide/mint-token/), with the difference of lite minting not being supported anymore.
 
+## Dumping & Loading
+
+To preserve your Devnet instance for future use, there are several options:
+- Dumping on exit (handles Ctrl+C, i.e. SIGINT, doesn't handle SIGKILL):
+```
+cargo run -- --dump-on exit --dump-path <PATH>
+```
+
+- Dumping after each transaction (done in background, doesn't block):
+```
+cargo run -- --dump-on transaction --dump-path <PATH>
+```
+
+### Loading
+
+To load a preserved Devnet instance, the options are:
+- Loading on startup:
+```
+cargo run -- --load-path <PATH>
+```
+
+Current load implementation is re-executing transactions stored in the dump file so the state of the blockchain can differ. For example, the timestamps of `StarknetBlock` will be different.
```

Comment:
True, it's more intuitive, fixed in 68a2a88243793ec097ebade098defbc2d73b9450, and about time-dependent functionality there is not much we can do about this now.

I remember that in python devnet we had some time manipulation features, maybe after a couple of months it will be introduced in devnet in rust, or maybe in the future, we will move from re-execution to dump state.

Reply:
yes, maybe we can add to docs this caveat note about time being changed

Reply:
but in the version that you suggested, it's self-explanatory:
"Conversely, loading is implemented as the re-execution of transactions from a dump. "

Reply:
Ok, you don't have to modify it. But if users come complaining about their timestamp logic not working, I'll tag you to explain :stuck_out_tongue:

Reply:
It's quick so I added this in 5a8c50aeca496656f0aa7c7baf25769b22282808